cortex-m-rt/src/lib.rs

```diff
@@ -536,24 +556,24 @@ cfg_global_asm! {
     "ldr r0, =__sbss
```

Comment:
As a(n optimization) note, you probably could skip the BSS zeroing if you are already zeroing the entire memory region. Something like `#[cfg(not(feature = "zero-init-ram"))]` above this block.

Reply:
Thanks, I added the suggested line.